### PR TITLE
[Task] Optimized get by UUID

### PR DESCRIPTION
**Pull request by lukmzig**

### CODE REVIEW

File: models/Tool/UUID/Dao.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function delete()
      */
     public function getByUuid($uuid)
     {
-        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME ." where uuid='" . $uuid . "'");
+        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME . ' where uuid= ?', [$uuid]);
```
Comment (Corepex) [inline models/Tool/UUID/Dao.php:83]:
```suggestion
        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME . ' where uuid= ?', [$this->db->quote($uuid)]);
```

Comment (Corepex) [reply models/Tool/UUID/Dao.php:83]: We should use `$this->db->quote()` here :)

**Review by Corepex — CHANGES_REQUESTED** [review]

File: models/Tool/UUID/Dao.php (hunk 1 above)
Comment (NiklasBr) [inline models/Tool/UUID/Dao.php]: I suggest supplying the third param with a type instead `[Types::STRING]` if you must:

> https://www.doctrine-project.org/projects/doctrine-dbal/en/2.13/reference/security.html#security

> Besides binding parameters you can also pass the type of the variable. This allows Doctrine or the underlying vendor to not only escape but also cast the value to the correct type.

Comment (jdreesen) [reply models/Tool/UUID/Dao.php]: Why? Is a prepared statement not enough?

**Pull request merged**